simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/configuration/SimulatorConfiguration.java

```diff
@@ -30,4 +31,27 @@
          */
         private int connectors;
     }
+
+    @Data
+    public static class WebSocketConfiguration {
+        /**
+         * Call timeout in milliseconds
+         */
+        private Long callTimeout;
```

Comment:
could you please specify time units in variable names? like readTimeoutMs, connectTimeoutMs?